docs/querying/basics.md

```diff
@@ -299,33 +331,34 @@ PromQL supports line comments that start with `#`. Example:
 
 ### Staleness
 
-When queries are run, timestamps at which to sample data are selected
-independently of the actual present time series data. This is mainly to support
+The timestamps at which to sample data, during a query, are selected
+independently of the actual present time series data. This method principally supports
```

Comment:
This statement about timestamps is not universally true, and so we should not make it here.
I think it's true for all range queries, and for instant queries that do not aggregate.

Separately, changing "mainly" to "principally" is unnecessary.

Reply:
Reading through the rest of the document and after watching https://www.youtube.com/watch?v=xIAEEQwUBXQ&t=497s the nth time I think we should just call out the fact that instant vector selectors and range vector selectors work differently and not try to explain here again, because that is redundant and can lead to faulty documentation later if things get out of sync, making proposal.